docs/content/Overlay.mdx

```diff
@@ -240,6 +240,13 @@ See the W3C accessibility recommendations for modals [here](https://www.w3.org/T
     }
   />
   <PropsTableSxRow />
+  <PropsTableRow
+    name="usePortal"
```

Comment:
`usePortal` makes me think it accepts a hook 🤔 Not sure what would be a better name though. Maybe `withPortal`? 🤷‍♂️

Reply:
good point!

how do you feel about `<Overlay inPortal={false}>` or `<Overlay insidePortal={false}>` instead of withPortal?

Reply:
good point!

how do you feel about these:

`<Overlay inPortal={false}>` or 
`<Overlay renderInPortal={false}>`

I'm tending towards the explicit `renderInPortal`

Reply:
I like `renderInPortal` 👍